src/mimefactory.rs

```diff
@@ -659,6 +615,92 @@ impl<'a> MimeFactory<'a> {
             encrypt_helper.should_encrypt(context, e2ee_guaranteed, &peerstates)?;
         let is_encrypted = should_encrypt && !force_plaintext;
 
+        let mut to = Vec::new();
```

Comment:
Why do we need to move this code block?

Reply:
This does reorder the headers though... I don't know how bad that is

Reply:
Can't really move it up as this creates borrow-problems